lineage/figures/figure8.py

```diff
@@ -1,55 +1,121 @@
 """
-File: figure7.py
-Purpose: Generates figure 7.
+File: figureS11.py
+Purpose: Generates figure S11.
 
 AIC.
 """
 import numpy as np
+from matplotlib.ticker import MaxNLocator
+
+from ..Analyze import run_Analyze_AIC
+from ..LineageTree import LineageTree
 
 # States to evaluate with the model
 from ..states.StateDistributionGaPhs import StateDistribution
 
-from .figureCommon import getSetup, subplotLabel
-from .figureS11 import run_AIC, figure_maker
+from .figureCommon import getSetup, lineage_good_to_analyze, subplotLabel
 
+# TODO
+# Calculate A-AIC where you removed the count of elements in T and pi that are zero, and add for those that are 1.
 
 desired_num_states = np.arange(1, 8)
 
+# Setting up state distributions and E
+Sone = StateDistribution(0.99, 0.9, 10, 2, 10, 2)
```

Comment:
Don't we have a common set of these in figureCommon?

Reply:
we only have 2 stateDistributions defined in figureCommon, here we need 4, and these parameters have to be well-chosen so that they are different from one another.